code/modules/projectiles/projectile/bullets.dm

```diff
@@ -62,13 +62,15 @@
 	fire_sound = 'sound/weapons/Gunshot_c20.ogg'
 
 /obj/item/projectile/bullet/midbullet/lawgiver
-	damage = 10
+	damage = 15
 	stun = 0
 	weaken = 0
 	superspeed = 1
 
 /obj/item/projectile/bullet/midbullet2
-	damage = 25
+	damage = 20
+	stun = 0
+	weaken = 0
```

Comment:
It doesn't matter if it would've been functionally the same. It would have had a different path, meaning you wouldn't mess with the functionality of all guns that currently fire midbullet2.